### ISSUE TITLE
problem with the private key

### ISSUE THREAD

**Issue by toutou8148** [kopanitsa/web3-arduino#22]
the value of the private key changes after forcing it to uint8_t.
In the provided example, the private key is declared as follow: 
const char PRIVATE_KEY[] = {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
                            0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
                            0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
                            0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00 };
in my case, I set it as follow:
const char PRIVATE_KEY[] = "0x.........";

In the provided example, the private key is forced to (uint8_t*). After forcing the private key, the value of the private key will be changed.

**Comment by itsHaseebSaeed** [comment]
Hi, there I'm having some issues running this library on my ESP32. Does it work on your esp32? If yes then I need your help. Please contact me using this email: haseebsyed30000@gmail.com. It would be huge favour

**Comment by toutou8148** [comment]
Yes, It works. The library contains some error. You should make some changes to make it work. Regarding the private key, you must set it in the following format:
const char PRIVATE_KEY[] = {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00 };

Please send me the error you get when setting the private key in that format and I will try to help you solving the problem.
